sbin/build.sh

```diff
@@ -56,50 +55,51 @@ function parseArguments() {
 }
 
 # Add an argument to the configure call
-addConfigureArg()
-{
+addConfigureArg() {
   # Only add an arg if it is not overridden by a user-specified arg.
   if [[ ${BUILD_CONFIG[CONFIGURE_ARGS_FOR_ANY_PLATFORM]} != *"$1"* ]] && [[ ${BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]} != *"$1"* ]]; then
     CONFIGURE_ARGS="${CONFIGURE_ARGS} ${1}${2}"
   fi
 }
 
 # Add an argument to the configure call (if it's not empty)
-addConfigureArgIfValueIsNotEmpty()
-{
+addConfigureArgIfValueIsNotEmpty() {
   # Only try to add an arg if the second argument is not empty.
   if [ ! -z "$2" ]; then
     addConfigureArg "$1" "$2"
   fi
 }
 
 # Configure the boot JDK
-configuringBootJDKConfigureParameter()
-{
+configureBootJDKConfigureParameter() {
   addConfigureArgIfValueIsNotEmpty "--with-boot-jdk=" "${BUILD_CONFIG[JDK_BOOT_DIR]}"
 }
 
+# Configure the Shenandoah GC build parameter for JDK 11-15
```

Comment:
Is it enabled by default on 16? Would be great if that was documented. Otherwise I've to google it every time. We should also record why <11 doesn't have it.